src/config.rs

```diff
@@ -344,6 +361,49 @@ mod tests {
             assert!(config.get_allowed_functions("app3").is_none());
         }
 
+        #[test]
+        fn is_function_allowed_returns_false_when_app_is_not_found() {
+            let config = create_test_config();
+            assert_eq!(
+                config.is_function_allowed("nopp", "zome_name", "fn_name"),
+                false
+            );
+        }
+
+        #[test]
+        fn is_function_allowed_returns_true_when_all_functions_allowed_for_app() {
```

Comment:
I'm noticing when reading this test, that the `is_function_allowed` method does not check whether a function exists at all for a given app. That's probably okay in the contexts in which the method is being used but I thought I'd mention it anyway.